crates/bevy_asset/src/asset_server.rs

```diff
@@ -361,6 +383,9 @@ impl AssetServer {
         Ok(asset_path_id)
     }
 
+    /// Queues the [`Asset`] at the provided path for loading and returns an untyped handle.
+    ///
+    /// See [`load`](#method.load).
```

Comment:
Oh huh, that's nice syntax 🤔

Reply:
Yeah, it's really nice that methods and such are defined as anchors in the html, and with a consistent syntax. Would be nice if cargo could link the anchor with just the square brackets though 🤔

Reply:
Actually, it should be avoided. Using the anchor directly makes it a standard markdown link, and as such rusdoc will not warn us if the link becomes dead, and it's dependent on rusdoc implementation (which probably won't change)
```suggestion
    /// See [`load`](AssetServer::load).
```